tests/integ/test_udf.py

```diff
@@ -733,6 +739,7 @@ def return_dict(v: dict) -> Dict[str, str]:
     )
 
 
+@pytest.mark.skipif(IS_IN_STORED_PROC, reason="need to support permanent udf")
```

Comment:
Let's change this to `Cannot create session in SP`. I think the original reason I put there is wrong 🤦

Reply:
changed